builtin/providers/aws/resource_aws_vpc_peering_connection.go

```diff
@@ -124,20 +124,38 @@ func resourceAwsVPCPeeringRead(d *schema.ResourceData, meta interface{}) error {
 			return nil
 		}
 	}
+	log.Printf("[DEBUG] VPC Peering Connection response: %#v", pc)
 
 	d.Set("accept_status", *pc.Status.Code)
-	d.Set("peer_owner_id", *pc.AccepterVpcInfo.OwnerId)
-	d.Set("peer_vpc_id", *pc.AccepterVpcInfo.VpcId)
-	d.Set("vpc_id", *pc.RequesterVpcInfo.VpcId)
```

Comment:
When using `d.Set`, it's recommend to simply reference the attribute and not dereference it. 

So instead of: 

    d.Set("vpc_id", *pc.RequesterVpcInfo.VpcId)

use 

    d.Set("vpc_id", pc.RequesterVpcInfo.VpcId)

`d.Set` will safely dereference it if it's nil, so it's considered safer than dereferencing its yourself. Note that this does _not_ work for `d.SetId`, which must be an value and not a pointer. 

Don't feel you must make this change, but I wanted to pass that on